src/control/lib/telemetry/telemetry.go

```diff
@@ -243,16 +244,16 @@ func (mb *metricBase) String() string {
 	return strings.TrimSpace(string(buf[:bytes.Index(buf, []byte{0})]))
 }
 
-func (sm *statsMetric) FloatMin() float64 {
-	return float64(sm.stats.dtm_min)
+func (sm *statsMetric) Min() uint64 {
+	return uint64(sm.stats.dtm_min)
 }
 
-func (sm *statsMetric) FloatMax() float64 {
-	return float64(sm.stats.dtm_max)
+func (sm *statsMetric) Max() uint64 {
+	return uint64(sm.stats.dtm_max)
 }
 
-func (sm *statsMetric) FloatSum() float64 {
-	return float64(sm.stats.dtm_sum)
+func (sm *statsMetric) Sum() uint64 {
+	return uint64(sm.stats.dtm_sum)
```

Comment:
Curious - For a long running system, is there any handling of Sum overflowing?

Reply:
It's up to the consumer to handle this. e.g. Prometheus just treats it the same way it does a restart, where the counter is reset to zero.